### PR TITLE
S3t3 styling

### PR DESCRIPTION
**Pull request by maxwell-01**

### CODE REVIEW

File: planet.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+$password = 'password';
+
+if(!isset($_GET['planetId'])){header('Location: ./index.php');}
+
+$db = new PDO($dsn, $username, $password);
+$planetId = $_GET['planetId'];
+$planet = PlanetHydrator::getPlanet($db, $planetId);
+
+
+
```
Comment (oritchie77) [inline planet.php]: just a couple of lines to remove here

File: planet.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300;900&display=swap" rel="stylesheet">
+</head>
+<body>
+<main>
+    <a class="button-container" href="./index.php#planet-<?=$planetId?>"><button type="button"><<object class="desktop-button-text"> Planets</object></button></a>
+    <h1><?=$planet->getName()?></h1>
+    <div class="planet-image-section">
+        <img src="./images/<?=$planet->getImageSrc()?>" alt="<?=$planet->getName()?>">
```
Comment (oritchie77) [inline planet.php]: could the alt be a little more descriptive?

File: planet.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <p><?=$planet->getSun()?></p>
+            <p class="detail-table-heading">Mass</p>
+            <p><?=$planet->getMassEarths()?> Earths</p>
+            <p class="detail-table-heading">Number of moons</p>
+            <p><?=$planet->getNumberOfMoons()?></p>
+            <p class="detail-table-heading">Mean surface temperature</p>
+            <p><?=$planet->getMeanSurfaceTemperatureC()?> C</p>
+        </div>
+    </div>
+
```
Comment (oritchie77) [inline planet.php]: few extra lines here

File: planet.scss [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+    &:hover {
+      background-color: #ffffff;
+      color: #000000;
+    }
+  }
+  .desktop-button-text {
+    display: none;
+  }
+
```
Comment (oritchie77) [inline planet.scss:49]: extra line

File: planet.scss [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+@media screen and (min-width: 800px) {
+  main {
+    display: grid;
+    column-gap: 20px;
+    grid-template-columns: 1fr 1fr;
+    .desktop-button-text {
+      display: inline;
+    }
+  }
+  .image-source-text {
```
Comment (oritchie77) [inline planet.scss:93]: add line here and a couple of others below

File: src/Planet.php [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    protected ?string $description;
+    protected ?string $image_description;
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+    public function getImageSrc(): string
+    {
+        return ($this->image_src === null) ? 'planet_not_found.png' : $this->image_src;
```
Comment (oritchie77) [inline src/Planet.php]: nice

File: src/Planet.php [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+        return $this->full_name;
+    }
+    public function getDescription(): string
+    {
+        return ($this->description === null) ? 'unknown' : $this->description;
+    }
+
+    public function getImageDescription(): string
+    {
+        return ($this->image_description === null) ? 'unknown' : $this->image_description;
```
Comment (oritchie77) [inline src/Planet.php:78]: do these need unit testing?

File: src/Planet.php [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+    public function setType(string $type): void
+    {
+        $this->type = $type;
+    }
+
+    public function createCardHtml(): string
+    {
+        return('
+        <div class="planet-card" id="planet-'.$this->id.'">
+            <a href="./planet.php?planetId='.$this->id.'"><img src="'.$this->image_src.'"></a>
```
Comment (oritchie77) [inline src/Planet.php]: I know this is from a previous story but should there be an alt for the image

**Review by oritchie77 — CHANGES_REQUESTED** [review]
overall really good, looks great, a couple of line spacing to correct, 

with the desktop display i thought that we were going to have a shaded background for the text ?

File: planet.php (hunk 1 above)
Comment (maxwell-01) [reply planet.php]: removed

File: planet.php (hunk 3 above)
Comment (maxwell-01) [reply planet.php]: removed

File: planet.scss (hunk 4 above)
Comment (maxwell-01) [reply planet.scss:49]: removed

File: planet.scss (hunk 5 above)
Comment (maxwell-01) [reply planet.scss:93]: done

File: src/Planet.php (hunk 7 above)
Comment (maxwell-01) [reply src/Planet.php:78]: Rayna said getters dont need unit testing :)

File: src/Planet.php (hunk 8 above)
Comment (maxwell-01) [reply src/Planet.php]: updated on story 2

**Pull request merged**